contracts/extensions/StreamingFeeSplitExtension.sol

```diff
@@ -34,8 +34,12 @@ import { IStreamingFeeModule } from "../interfaces/IStreamingFeeModuleV2.sol";
  * @title StreamingFeeSplitExtension
  * @author Set Protocol
  *
- * Smart contract global extension which provides DelegatedManager owner and methodologist the ability to accrue and split 
+ * Smart contract global extension which provides DelegatedManager owner and methodologist the ability to accrue and split
  * streaming fees. Owner may configure the fee split percentages.
+ *
+ * Notes
+ * - the fee split is set on the Delegated Manager contract
+ * - when fees distributed via this contract will be inclusive of all fee types
```

Comment:
Note the `streamingFeeSplitExtension` performs an `accrueFeeAndDistribute`.

So it collects both fees previously accrued (like issuance and redemption fees which are accrued on `issue`/`redeem` calls) and fees pending accrual (like streaming fees).